lib/configise.js

```diff
@@ -27,21 +27,33 @@ var user = process.env.USER;
 var env = process.env.ENV;
 var node_env = process.env.NODE_ENV;
 
+//check for "' warn user and remove it
+_(["USER", "ENV", "NODE_ENV"]).each(function(env) {
+  if ((/['||"]/g).test(process.env[env])) {
+    console.warn("found quotes around " + env + " removing it");
+    process.env[env].replace(/['||"]/, "");
```

Comment:
I'm pretty sure that's not the correct regex, either on line 32 or 34. You want the difficult-to-read `/['"]/g`.